dataactvalidator/config/sqlrules/d22_detached_award_financial_assistance.sql

```diff
@@ -0,0 +1,11 @@
+-- AwardingAgencyCode must be a valid 3-digit CGAC agency code.
+SELECT
+    dafa.row_number,
+    dafa.awarding_agency_code
+FROM detached_award_financial_assistance as dafa
+WHERE dafa.submission_id = {0}
+    AND (dafa.awarding_agency_code != ''
```

Comment:
You don't need this outer set of parenthesis. You probably also don't need to check for '' because that situation will cause an error earlier in the process anyway.